### DISCUSSION TITLE
RDKit and Pandas in Jupyter notebook

### DISCUSSION THREAD

**Discussion by ustojiljkoff** [rdkit/rdkit#7080] [Q&A] [answered]
Hello everyone,

I have an issue with using PandasTools in Jupyter notebook. When I use line:
```
PandasTools.AddMoleculeColumnToFrame(df, "SMILES", "Structure")
```
I get a `Structure` column, but rows contain output similar as follows:
```
<rdkit.Chem.rdchem.Mol object at 0x0000022EE850E500>
```

When I do the same thing in GoogleColab, I don't have any issues, and structures are, in fact, in a dataframe. Did anyone face the same issue and managed to find a solution? I have tried some options, but never managed to solve it.

Cheers,
Uros

**Comment by ptosco** [comment, marked as answer]
@ustojiljkoff You may switch on molecule image visualization on a per-DataFrame basis with:
`PandasTools.ChangeMoleculeRendering(df)`
or for all DataFrames in your Jupyter session with:
`PandasTools.RenderImagesInAllDataFrames()`
